src/chatdlg.cpp

```diff
@@ -106,8 +106,34 @@ void CChatDlg::OnClearChatHistory()
 void CChatDlg::AddChatText ( QString strChatText )
 {
     // add new text in chat window
-    txvChatWindow->append ( strChatText );
-
-    // notify accessibility plugin that text has changed
-    QAccessible::updateAccessibility ( new QAccessibleValueChangeEvent ( txvChatWindow, strChatText ) );
+	
+    // analyze strChatText to check if hyperlink (limit ourselves to https:://)
+	int indx_http  = strChatText.indexOf("https://", 0);
+	if (indx_http != -1)
+	{
+		int indx_space = strChatText.indexOf(" ", indx_http); 
+		if (indx_space==-1) indx_space=strChatText.length();
+
+		int cutslash = (strChatText.at(indx_space-1)=='/') ? 1:0;		// drop "/" if last character of url text
+		
+		QString URL_name = strChatText.mid(indx_http,indx_space-indx_http-cutslash); // as entered by the user
+		QUrl URL = QUrl::fromUserInput(URL_name);
```

Comment:
Nice.  With the check for http/https, this should mean at least half-way sane values go in.  If it comes back invalid, the check below then catches it.  I'd still like the text more readable than "normal" so it catches the reader attention and is clearer.